test.ini

```diff
@@ -9,7 +9,7 @@ host = 0.0.0.0
 port = 5000
 
 [app:main]
-use = config:../ckan/test-core.ini
```

Comment:
Don't change this path, it's the standard on a source install, instead update the path in the GA workflow like in here:

https://github.com/ckan/ckanext-geoview/blob/9ddc29cfcdb66c20109eef82eb06b04c65fdb69b/.github/workflows/test.yml#L53